### PR TITLE
Build stat var search index for custom DC data

### PR DESCRIPTION
**Pull request by shifucun**
Setting SearchNames in StatVarGroupNode_ChildSV protobuf does the job.

### CODE REVIEW

File: internal/server/statvar/fetcher/svg_fetcher.go [hunk 1]
```diff
@@ -165,7 +165,11 @@ func FetchAllSVG(
 			}
 			result[svg].ChildStatVars = append(
 				result[svg].ChildStatVars,
-				&pb.StatVarGroupNode_ChildSV{Id: sv, DisplayName: name},
+				&pb.StatVarGroupNode_ChildSV{
+					Id:          sv,
+					DisplayName: name,
+					SearchNames: []string{name},
```
Comment (keyurva) [inline internal/server/statvar/fetcher/svg_fetcher.go:171]: Do you have access to the SV descriptions? If so, consider adding it to `SearchNames`.

Also, it'll be good to start adding tests for custom dc functionality.

**Review by keyurva — APPROVED** [review]

File: internal/server/statvar/fetcher/svg_fetcher.go (hunk 1 above)
Comment (shifucun) [reply internal/server/statvar/fetcher/svg_fetcher.go]: added description and golden tests

File: internal/server/statvar/fetcher/svg_fetcher.go [hunk 2]
```diff
@@ -165,7 +168,11 @@ func FetchAllSVG(
 			}
 			result[svg].ChildStatVars = append(
 				result[svg].ChildStatVars,
-				&pb.StatVarGroupNode_ChildSV{Id: sv, DisplayName: name},
+				&pb.StatVarGroupNode_ChildSV{
+					Id:          sv,
+					DisplayName: name,
+					SearchNames: []string{name, description},
```
Comment (keyurva) [inline internal/server/statvar/fetcher/svg_fetcher.go:174]: Consider checking that `description` has a value before adding it.

Comment (shifucun) [reply internal/server/statvar/fetcher/svg_fetcher.go]: done

**Pull request merged**